personio-recruiting-api.yaml

```diff
@@ -702,18 +702,18 @@ components:
           type: string
           maxLength: 100
         'categorised_documents[n][file]':
-          description: 'Nth document <br/> The file should be an upload stream. <br/> You can check a request example in the "A note on categorised documents" section. <br/> [array of files] <br/> allowed extensions are ''pdf'', ''docx'', ''doc'', ''jpg'', ''png'', ''txt'', ''jpeg'', ''odt'', ''ods'',''xlsx'', ''rtf'', ''xls'', ''pptx'', ''ppt'', ''gif'', ''tif'', ''tiff'', ''bmp'', ''csv'', ''rar'', ''gz'', ''zip'', ''7z''; <br/> filesize per document < 20M; <br/> total post size < 64M; <br/>'
```

Comment:
All of these fields only describe the legacy monolith endpoints, not the ones in SAA. Those are configured here: https://github.com/personio/api-docs/pull/83/files#diff-d298334657cbc906671f5ba2d83411e8ee529e2d4e17c52486afd7bdf67dc75aL438